libs/components/forms/src/lib/modules/radio/radio.component.ts

```diff
@@ -58,46 +58,52 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
   public set checked(value: boolean) {
     const newCheckedState = !!value;
 
-    if (this._checked !== newCheckedState) {
-      this._checked = newCheckedState;
-      this._checkedChange.next(this._checked);
+    if (this.#_checked !== newCheckedState) {
+      this.#_checked = newCheckedState;
+      this.#_checkedChange.next(this.#_checked);
 
       if (newCheckedState) {
         this.selectedValue = this.value;
       }
     }
 
-    this.changeDetector.markForCheck();
+    this.#changeDetector.markForCheck();
   }
 
   public get checked(): boolean {
-    return this._checked;
+    return this.#_checked;
   }
 
   /**
    * Indicates whether to disable the input.
    * @default false
    */
   @Input()
-  public set disabled(value: boolean) {
+  public set disabled(value: boolean | undefined) {
     const coercedValue = SkyFormsUtility.coerceBooleanProperty(value);
     if (coercedValue !== this.disabled) {
-      this._disabled = coercedValue;
-      this._disabledChange.next(this._disabled);
-      this.changeDetector.markForCheck();
+      this.#_disabled = coercedValue;
+      this.#_disabledChange.next(this.#_disabled);
```

Comment:
We're accessing a setter's property outside of the setter. Maybe this should be a new value.
```
#disabledChange = new Subject<boolean>();

constructor() {
  #disabledChangeObs = this.#disabledChange.asObservable();
}

public get disabledChange(): Observable<boolean> {
  return this.#disabledChangeObs;
}
```
For example: https://github.com/blackbaud/skyux/blob/main/libs/components/core/src/lib/modules/affix/affixer.ts#L48

Reply:
Fixed